public/Face.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { faceToSvgString } from "../src/faceToSvgString";
+import { FaceConfig, Overrides } from "../src/types";
+import { objStringifyInOrder } from "./utils";
+import { useInView } from "react-intersection-observer";
+
+/*
+    This component is responsible for rendering the face SVG string
+    It uses the faceToSvgString function to generate the SVG string
+    It requires the faceConfig to be set, with the overrides being optional
+    The width and className are also optional
+    The SVG string is then rendered in a div element
+    The SVG string is set as the innerHTML of the div element
+*/
+
+export const Face = ({
+  faceConfig,
+  overrides,
+  maxWidth,
+  width,
+  className,
+  lazyLoad,
+}: {
+  faceConfig: FaceConfig;
+  overrides?: Overrides;
+  maxWidth?: number;
+  width?: number;
+  className?: string;
+  lazyLoad?: boolean;
+}) => {
+  const [ref, inView] = useInView({
+    triggerOnce: true,
+    threshold: 0,
+  });
```

Comment:
Is this doing what we want?

Imagine you scroll all the way everywhere, so all SVGs get loaded. Then you change a feature. That will then re-render all the SVGs, right? But we don't want that, we want only the visible ones to update. The ones offscreen should either remain static until they come in screen, or maybe even be deleted (replaced by a placeholder empty div) when off screen, if that matters for performance.

Reply:
Switched to `triggerOnce: false` so it'll detach upon scroll out of view